web/.env.production

```
@@ -0,0 +1,2 @@
+VITE_PUBLIC_POSTHOG_PROJECT_TOKEN=phc_wbWWQQNfpx4MvNLoJ3CKcnhVaAp625urDu39YAvusGJe
+VITE_PUBLIC_POSTHOG_HOST=https://us.i.posthog.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_PUBLIC_POSTHOG_HOST key should go before the VITE_PUBLIC_POSTHOG_PROJECT_TOKEN key

(UnorderedKey)